assets/templates/dining_lists/dining_slot_diners.html

```diff
@@ -82,29 +94,30 @@
                                             </label>
                                         {% else %}
                                             <div class="col-3">
-                                                {% if entry.has_shopped %} Shop {% endif %}
+                                                {% if entry.has_shopped %}Shop{% endif %}
                                             </div>
                                             <div class="col-3">
-                                                {% if entry.has_cooked %} Cook {% endif %}
+                                                {% if entry.has_cooked %}Cook{% endif %}
                                             </div>
                                             <div class="col-3">
-                                                {% if entry.has_cleaned %} Clean {% endif %}
+                                                {% if entry.has_cleaned %}Clean{% endif %}
                                             </div>
                                         {% endif %}
                                     {% else %}
+                                        {# External entries have no shop/cook/clean stats #}
                                         <div class="col-9"></div>
                                     {% endif %}
-                                    {% if can_edit_pay %}
+                                    {% if dining_list|is_owner:user %}
```

Comment:
If the right to edit payments and cooking stats is the same, there is no need any more for the current seperation and the check can occur only once. Ie. If owner: display 4 clickable buttons, else, display div results.
That said, currently the shop, cook and clean stats can be altered up to 2 days after teh dining list (in order to prevent abuse of help percentage through old dining lists). However, payment is open longer as some people are notoriously slow in paying shit. Maybe we can incorporate that in the layout?

Reply:
Yes sounds good. Won't do it in this PR though

Reply:
That's fine, maybe I'll do it some day.